src/oscar/apps/customer/migrations/0004_auto_20170413_1853.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-04-13 17:53
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('customer', '0003_update_email_length'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Please can you rebuild these migrations? Seems like you changed the `email` field in between this migration and the next one.